package.json

```diff
@@ -1347,7 +1378,7 @@
 				},
 				{
 					"command": "pr.copyVscodeDevPrLink",
-					"when": "github:inReviewMode"
+					"when": "github:inReviewMode && remoteName != codespaces"
```

Comment:
Yeah, the main thing is ensuring that vscode.dev links don't show up in github.dev, and since Codespaces are served from the github.dev domain I think the same expectation applies for vscode.dev links to Codespaces.